packages/commonwealth/server/routes/createChain.ts

```diff
@@ -155,12 +157,13 @@ const createChain = async (
       privateUrl = node.private_url;
     }
 
-    const existingChain = await models.Chain.findOne({
-      where: { address: req.body.address, chain_node_id: node.id }
-    });
-    if (existingChain) {
-      return next(new Error(Errors.ChainAddressExists));
-    }
+    // TODO: Document to BD that multiple communities with the same contract can now exist
+    // const existingChain = await models.Chain.findOrCreate({
+    //   where: { address: req.body.address, chain_node_id: node.id }
```

Comment:
Do we protect against creating duplicate Cosmos/Substrate chains?

Reply:
I don't believe so, but this check by using `req.body.address` wasn't doing so either

Reply:
It would be, because it'll look only for duplicate chain nodes if address is empty